Mathlib/RingTheory/TwoSidedIdeal/Basic.lean

```diff
@@ -43,6 +43,9 @@ section NonUnitalNonAssocRing
 
 variable {R : Type*} [NonUnitalNonAssocRing R] (I : TwoSidedIdeal R)
 
+lemma ringCon_injective : Function.Injective (TwoSidedIdeal.ringCon (R := R)) := by
```

Comment:
Could you add the `Equiv` too, or maybe an `OrderIso`?

Reply:
done